core/src/main/java/com/graphhopper/routing/lm/LandmarkSuggestion.java

```diff
@@ -40,7 +40,7 @@ public BBox getBox() {
      * to specify an explicit bounding box. TODO: support GeoJSON instead.
      */
     public static final LandmarkSuggestion readLandmarks(String file, LocationIndex locationIndex) throws IOException {
-        // landmarks should be suited for all vehicles
+        // landmarks should be suited for all profiles
```

Comment:
You mean public transit? In its current form A* should work and so also landmarks but not sure if it works nor if it makes it faster